db/migrate/20240304161254_create_plants.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePlants < ActiveRecord::Migration[7.1]
+  def change
+    create_table :plants do |t|
+      t.string :name
+      t.string :description
+      t.integer :days_to_harvest
+      t.references :plot, null: false, foreign_key: true
```

Comment:
You don't need line 7, but instead would make a plot_plants table with a foreign key for plot and a foreign key for plant